cli/Cargo.toml

```diff
@@ -39,7 +39,7 @@ http = "1.3.1"
 # git pending:
 # - https://github.com/alexliesenfeld/httpmock/issues/127
 # - https://github.com/alexliesenfeld/httpmock/pull/131
-httpmock = { git = "https://github.com/flox/httpmock.git", branch = "flox/remove_recording_struct", default-features = false, features = [
+httpmock = { git = "https://github.com/flox/httpmock.git", branch = "flox/better-errors", default-features = false, features = [
```

Comment:
It's going to get really weird if we stack up feature branches on top of each other so I merged Dan's changes https://github.com/flox/httpmock/pull/1 and PRed yours https://github.com/flox/httpmock/pull/2. Can we merge that and use `master` here?

Reply:
Totally, there was no indication why we were keeping a branch around so I just stuck with that pattern.

Reply:
I just pushed a commit here that points at the `master` branch, and the PR on the `httpmock` side is merged.